CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changes by Version
 ==================
 
+* add METRICS_STORAGE_TYPE for metrics query
```

Comment:
I don't think you need this, there's a Makefile target to generated the changelog from git commits: https://github.com/jaegertracing/jaeger-operator/blob/main/RELEASE.md#generating-the-changelog